client/src/util/auth.jsx

```diff
@@ -30,243 +42,228 @@ function useAuthProvider() {
   // Store auth user in state
   // `user` will be an object, `null` (loading) or `false` (logged out)
   const [user, setUser] = useState(null);
-
-  // Merge extra user data from the database
-  // This means extra user data (such as payment plan) is available as part
-  // of `auth.user` and doesn't need to be fetched separately. Convenient!
-  let finalUser = useMergeExtraData(user, { enabled: MERGE_DB_USER });
-
-  // Add custom fields and formatting to the `user` object
-  finalUser = useFormatUser(finalUser);
+  console.log(user);
 
   // Handle response from auth functions (`signup`, `signin`, and `signinWithProvider`)
-  const handleAuth = async (response) => {
-    const {
-      data: { user },
-    } = response;
-
-    // If email is unconfirmed throw error to be displayed in UI
-    // The user will be confirmed automatically if email confirmation is disabled in Supabase settings
-    if (!user.email_confirmed_at) {
-      throw new Error(
-        "Thanks for signing up! Please check your email to complete the process.",
-      );
-    }
+  //!whats the point of this
+  const handleAuth = async (user) => {
```

Comment:
Remove the comment and rename this function to `storeUser` if you are confused by the function name

Reply:
Removed comment, let function name and function alone in case in future there is more logic to be handled when authorizing users